web/src/styles/operator-shell.css

```
@@ -4014,3 +4094,48 @@
+   tokens to the operator palette within the .opr-build-activity wrapper so the
+   thinking + tool-call trace blends into the terminal-muted surface. Scoped to
+   the wrapper, so the office feed is untouched. */
+.opr-build-activity .app-build-activity {
+  --border: var(--t-line);
+  --bg-card: var(--t-surface);
+  --text: var(--t-ink);
+  --text-secondary: var(--t-dim);
+  --accent: var(--t-accent);
+  --green: var(--t-green);
+  --red: var(--t-red);
+  border: 1px solid var(--t-line);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)